cmd/akamai-purger/main.go

```diff
@@ -126,22 +135,32 @@ func main() {
 		log:    logger,
 	}
 
-	stop, stopped := make(chan bool, 1), make(chan bool, 1)
+	stop, stopped := make(chan bool), make(chan bool)
```

Comment:
What does making `stopped` buffered get us here? It's not necessarily bad, but introducing blocking writes doesn't really do anything since it just prolongs exiting from a hanging goroutine.

Reply:
You're right - I got confused while writing this patch and thought we were exiting one of the receives early because it was unbuffered. But of course receiving from an empty channel will block regardless of buffering. I reverted to what it was before.